hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestNSSummaryTaskWithFSO.java

```diff
@@ -117,6 +121,9 @@ private TestNSSummaryTaskWithFSO() {
 
   @BeforeClass
   public static void setUp() throws Exception {
+    ozoneConfiguration = new OzoneConfiguration();
+    ozoneConfiguration.setLong(OZONE_RECON_NSSUMMARY_FLUSH_TO_DB_MAX_THRESHOLD,
+        15 * 1000L);
```

Comment:
15k is too big for this test since it won't have 15k objects.  Just have tried with 10 and it worked well!  Please use 10.  Otherwise, the change won't be tested.

Reply:
Ok set as 10.. Pls re-review @szetszwo

Reply:
@szetszwo , pls review